at_client/lib/src/client/at_client_impl.dart

```diff
@@ -79,15 +80,18 @@ class AtClientImpl implements AtClient {
     _atClientInstanceMap[currentAtSign] = atClientImpl;
   }
 
-  /// use [create]
+  @Deprecated("Use [create]")
   AtClientImpl(
       String _atSign, String? namespace, AtClientPreference preference) {
     currentAtSign = AtUtils.formatAtSign(_atSign);
     _preference = preference;
     _namespace = namespace;
   }
 
-  static Future<AtClient> create(String currentAtSign, String? namespace,
+  static Future<AtClient> create(
+      AtClientManager atClientManager,
+      String currentAtSign,
+      String? namespace,
       AtClientPreference preferences) async {
```

Comment:
For backwards compatibility, you could make atClientManager an optional param and if it's not supplied use the current static mechanism to get the instance

Reply:
done